main/network.go

```diff
@@ -229,44 +228,6 @@ func tcpNMEAOutListener() {
 	}
 }
 
-
-/* Server that can be used to feed NMEA data to, e.g. to connect OGN Tracker wirelessly */
-func tcpNMEAInListener() {
```

Comment:
This is now considered it's own 'GPS' device and can be turned on/off (for security aswell..). This is to ensure we can still receive flarm over TCP will not processing location/time information that could be conflicting.